Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/EditorMeshComponent.cpp

```diff
@@ -76,6 +76,7 @@ namespace AZ
                             ->DataElement(AZ::Edit::UIHandlers::Default, &MeshComponentConfig::m_sortKey, "Sort Key", "Transparent meshes are first drawn by sort key, then depth. Use this to force certain transparent meshes to draw before or after others.")
                             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &MeshComponentConfig::m_excludeFromReflectionCubeMaps, "Exclude from reflection cubemaps", "Model will not be visible in baked reflection probe cubemaps")
                                 ->Attribute(AZ::Edit::Attributes::ChangeNotify, Edit::PropertyRefreshLevels::ValuesOnly)
+                            ->DataElement(AZ::Edit::UIHandlers::CheckBox, &MeshComponentConfig::m_isAlwaysDynamic, "Always Moving", "Forces this mesh to be considered to always be moving, even if the transform didn't update. Useful for meshes with vertex shader animation.")
```

Comment:
Eventually it would be nice to be able to drive the always-dynamic behavior automatically via the material in some way. But I do think this will remain useful as a fallback for a user to be able to say 'hey, I know this needs to be re-rendered every frame, and the automatic systems aren't picking it up correctly'.

I do think this probably belongs a little further down in the UI below IsRayTracingEnabled, since it's a less-often used feature.

Reply:
I agree about the material thing - was actually chatting with Brendan about that. I think that'll just take a bit more work to figure out the right way to do though. I need to chat with Chris about that while he's still here. 

As for where the property shows up in the UI, I'll move it down to just above LOD stuff.